project/src/com/google/daggerqueryui/tests/binding_graph_test.js

```diff
@@ -0,0 +1,122 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+var assert = chai.assert;
+
+describe('Binding Graph', function() {
+  it('adds one edge correctly', function() {
+    bindingGraph.clear();
```

Comment:
This same line is being called in every test. You could consider moving this into a beforeEach hook. https://mochajs.org/#hooks

Reply:
Thanks, [done](https://github.com/googleinterns/dagger-query/pull/41/commits/9d7ddf8a267f15c62783fca633714e2702be938d) 🍰